index.d.ts

```diff
@@ -12,6 +12,7 @@ declare module 'sw-toolbox' {
     name: string
     maxEntries?: number
     maxAgeSeconds?: number
+    queryOptions?: CacheQueryOptions
```

Comment:
@mohsen1, can you confirm that this change looks good from a TypeScript perspective? `CacheQueryOptions` is a [standard interface](https://w3c.github.io/ServiceWorker/#dictdef-cachequeryoptions), and I'm not sure if just referencing it like this is sufficient, or whether it needs to be defined as a stub interface inside this file.